cmake/find_python_module.cmake

```diff
@@ -0,0 +1,35 @@
+#[[[
```

Comment:
When you have separate files meant for inclusion best practice is to start them with an [`include_guard()`](https://cmake.org/cmake/help/latest/command/include_guard.html?highlight=include_guard) command. This is for the same reason as the C/C++ header guards.